consensus/progpow/algorithm.go

```diff
@@ -37,17 +37,17 @@ import (
 )
 
 const (
-	datasetInitBytes   = 1 << 30    // Bytes in dataset at genesis
-	datasetGrowthBytes = 1 << 23    // Dataset growth per epoch
-	cacheInitBytes     = 1 << 24    // Bytes in cache at genesis
-	cacheGrowthBytes   = 1 << 17    // Cache growth per epoch
-	epochLength        = 2147483647 // Blocks per epoch
-	mixBytes           = 128        // Width of mix
-	hashBytes          = 64         // Hash length in bytes
-	hashWords          = 16         // Number of 32 bit ints in a hash
-	datasetParents     = 256        // Number of parents of each dataset element
-	cacheRounds        = 3          // Number of rounds in cache production
-	loopAccesses       = 64         // Number of accesses in hashimoto loop
+	datasetInitBytes   = 1 << 30 // Bytes in dataset at genesis
+	datasetGrowthBytes = 1 << 23 // Dataset growth per epoch
+	cacheInitBytes     = 1 << 24 // Bytes in cache at genesis
+	cacheGrowthBytes   = 1 << 17 // Cache growth per epoch
+	epochLength        = 4320    // Blocks per epoch
```

Comment:
probably need to rethink this number as @kiltsonfire recently changed the prime time